opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -161,82 +161,49 @@ message MetricDescriptor {
   // type is the type of values this metric has.
   Type type = 4;
 
-  // Temporality is the temporal quality values of a metric have. It
-  // describes how those values relate to the time interval over which they
-  // are reported.
-  enum Temporality {
-    // INVALID_TEMPORALITY is the default Temporality, it MUST not be
-    // used.
-    INVALID_TEMPORALITY = 0;
-
-    // INSTANTANEOUS is a metric whose values are measured at a particular
-    // instant. The values are not aggregated over any time interval and are
-    // unique per timestamp. As such, these metrics are not expected to have
-    // an associated start time.
-    INSTANTANEOUS = 1;
-
-    // DELTA is a metric whose values are the aggregation of measurements
-    // made over a time interval. Successive metrics contain aggregation of
-    // values from continuous and non-overlapping intervals.
-    //
-    // The values for a DELTA metric are based only on the time interval
-    // associated with one measurement cycle. There is no dependency on
-    // previous measurements like is the case for CUMULATIVE metrics.
-    //
-    // For example, consider a system measuring the number of requests that
-    // it receives and reports the sum of these requests every second as a
-    // DELTA metric:
+  // Structure describes the relation between data of a Metric. This
+  // relation is expressed by the operation(s) supported by the data. The
+  // Structure of data is important to know if further analysis or roll-up
+  // of the data is to be performed. Without knowledge of data Structure
+  // meaningless results could be produced from these tasks.
+  //
+  // Note:
+  // Knowing the Structure of data does not guarantee meaningfully results
+  // for arbitrary operations on the data, rather it means that the results
+  // of performing the operation(s) related to the data Structure COULD
+  // produce meaningful results.
+  //
+  // Take for example adding together two different the measurements of
+  // request counts from a server. This data has ADDING structure, but if
+  // the two measurements were for different services, or even different
+  // servers, this result is likely meaningless.
```

Comment:
I'm not sure I agree. It the metric counts requests, it's meaningful to combine them in as much as you get a combined count, which is well defined. If the metric is the same, it should be a meaningful aggregation: it's a count of requests from multiple servers, I think.

The best example, for me, of a structural property is with measurements from a GROUPING instrument. If you have one individual measurement, there is nothing you can combine with it to make it into a different individual measurement.  Combining measurements with a GROUPING instruments gives you more measurements.  (Not sure that helps, but it might be a better example?)

Reply:
Removed.